portaudio will fail pkgconf test pipeline

If the pkgconf test pipeline is added to portaudio it will fail in the following way:

```
2024/11/12 23:14:08 INFO running step "test/pkgconf"
2024/11/12 23:14:08 WARN + '[' -d /home/build ] uses=test/pkgconf
2024/11/12 23:14:08 WARN + cd /home/build uses=test/pkgconf
2024/11/12 23:14:08 WARN + exit 0 uses=test/pkgconf
2024/11/12 23:14:08 INFO running step "pkgconf build dependency check" uses=test/pkgconf
2024/11/12 23:14:08 WARN + '[' -d /home/build ] uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + cd /home/build uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + basename /home/build/melange-out/portaudio-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + dev_pkg=portaudio-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + cd / uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + apk info -L portaudio-dev uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + grep '\.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN WARNING: opening /home/user/src/wolfi-os/packages: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN WARNING: opening from cache https://packages.wolfi.dev/os: No such file or directory uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + grep -q ^Name: usr/lib/pkgconfig/portaudio-2.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + basename usr/lib/pkgconfig/portaudio-2.0.pc .pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + lib_name=portaudio-2.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + echo usr/lib/pkgconfig/portaudio-2.0.pc uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + grep -q '^usr/lib/pkgconfig/portaudio-2.0.pc$\|^usr/share/pkgconfig/portaudio-2.0.pc$' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + pkgconf --exists portaudio-2.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN + pkgconf --print-errors --exists portaudio-2.0 uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN Package alsa was not found in the pkg-config search path. uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN Perhaps you should add the directory containing `alsa.pc' uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN to the PKG_CONFIG_PATH environment variable uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 WARN Package 'alsa', required by 'portaudio-2.0', not found uses=test/pkgconf name="pkgconf build dependency check"
2024/11/12 23:14:08 INFO pod 82524f8d2e3d3e374a990a50c01725696d144db907720ec6d3bf5bb9440d4c68 terminated
2024/11/12 23:14:08 INFO ERROR: failed to test package. the test environment has been preserved:
2024/11/12 23:14:08 INFO   workspace dir: /home/user/tmp/melange-workspace-4002088622
2024/11/12 23:14:08 INFO   guest dir: /home/user/tmp/melange-guest-2459058349
2024/11/12 23:14:08 ERRO failed to test package: unable to run pipeline: unable to run pipeline: unable to run pipeline: task exited with code 1
make: *** [Makefile:147: test/portaudio] Error 1
```

--- a/portaudio.yaml
+++ b/portaudio.yaml
@@ -1,7 +1,7 @@
 package:
   name: portaudio
   version: 19.7.0
-  epoch: 0
+  epoch: 1
   description: "A cross-platform, open-source C language library for real-time audio input and output"
   copyright:
     - license: MIT
@@ -62,7 +62,11 @@ subpackages:
       - uses: split/dev
     dependencies:
       runtime:
+        - alsa-lib-dev
         - portaudio
+    test:
+      pipeline:
+        - uses: test/pkgconf
 
 update:
   enabled: true

